<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModbusWriteMultipleCoilsDialog</class>
 <widget class="QDialog" name="ModbusWriteMultipleCoilsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>591</width>
    <height>517</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>591</width>
    <height>517</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Slave ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="box_slave_id">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="6">
    <widget class="QListWidget" name="list_coils"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="button_send">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="box_address">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="button_cancel">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Quantity:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="box_quantity">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="button_update_coil_list">
     <property name="text">
      <string>Update Coil List</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="button_set_all">
     <property name="text">
      <string>Set All</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="button_clear_all">
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
